### DISCUSSION TITLE
Multiselect - How to enabled ?

### DISCUSSION THREAD

**Discussion by nikitakofman** [prevwong/craft.js#689] [Q&A]
Hello, 

I've seen in the documentation that from the 0.2 version the multiselect was implemented. 

I am not finding how to activate it, i've tried:

 handlers={(store) =>
                          new CustomEventHandlers({
                            store,
                            isMultiSelectEnabled: () => true,
                            removeHoverOnMouseleave: true,
                          })
                        }

In the editor, but it does nothing.

Thanks in advance for your help !

**Comment by prevwong** [comment]
By default, you need to hold down the meta key (the Command key on MacOS) and click to select multiple elements. On Windows, the meta key is the Windows key, so you probably want to change it:

```tsx
handlers={(store) =>
  new CustomEventHandlers({
    store,
    isMultiSelectEnabled: (e) => !!e.ctrlKey, // activate when Control key is pressed
  })
}
```

**Reply by nikitakofman** [reply]
Thank you !
